integration-tests/src/test/resources/domain-home-on-pv/create-domain.py

```diff
@@ -106,6 +106,13 @@ def getEnvVar(var):
   cmo.setCluster(cl)
   print('Done setting attributes for Server Template: %s' % templateName);
 
+  templateChannelName = cluster_name + "-NAP"
+  create(templateChannelName, 'NetworkAccessPoint')
+  cd('NetworkAccessPoints/%s' % templateChannelName)
+  set('PublicPort', server_port + 10)
+  # set('PublicAddress', 'junkvalue')
+  set('ListenAddress', '%s-%s${id}' % (domain_uid, managed_server_name_base_svc))
```

Comment:
It's not necessary to set the ListenAddress (same holds true for both domain11 and domain12).

Reply:
Actually it might be best, for test purposes, to set the ListenAddress to a completely invalid value 'not-a-real-dns-address' or some-such.   This would help verify that the automatic situational config generated by the introspector properly overrides the NAP address.

Reply:
I'll fix that. Thanks!

Reply:
Cool.  Once you do this, you can 'ad-hoc' check if the override takes effect at least at the mbean level using wlst.sh per the debugging section of 'config-overrides.md'.   Also, if you add a 'curl' test, as I'm recommending, that also implicitly tests if the override is working - as if it isn't, the curl couldn't succeed...